types/src/v0/error.rs

```diff
@@ -0,0 +1,104 @@
+use core::fmt::Debug;
+
+use jf_merkle_tree::MerkleTreeError;
+use serde::{Deserialize, Serialize};
+use snafu::Snafu;
+use thiserror::Error;
+
+use super::{BlockSize, FeeAmount, NsTableValidationError};
+use crate::{BlockMerkleCommitment, FeeMerkleCommitment};
+
+#[derive(Clone, Debug, Snafu, Deserialize, Serialize)]
+pub enum Error {
+    // TODO: Can we nest these errors in a `ValidationError` to group them?
```

Comment:
Not sure, where did this `Error` come from? What is it meant to represent? I don't remember seeing it before.

Reply:
Looks like this was previously found in `lib.rs` and this `TODO` was already present.

Reply:
I'm thinking we ultimately don't need these global errors, and we should create an issue to remove this in the future, localize there errors to the modules where they are being used, and replace snafu w/ thiserror.

Reply:
Yeah, we can remove this in a separate PR.

Reply:
new issue: https://github.com/EspressoSystems/espresso-sequencer/issues/1681